### ISSUE TITLE
Flaky-test: BrokerBkEnsemblesTests.testCrashBrokerWithoutCursorLedgerLeak

### ISSUE THREAD

**Issue by lhotari** [apache/pulsar#13625]
[BrokerBkEnsemblesTests is flaky. The testCrashBrokerWithoutCursorLedgerLeak test method fails sporadically.

[example failure](https://github.com/apache/pulsar/runs/4713066229?check_suite_focus=true#step:9:482)

```
Error:  Tests run: 13, Failures: 4, Errors: 0, Skipped: 9, Time elapsed: 9.915 s <<< FAILURE! - in org.apache.pulsar.broker.service.BrokerBkEnsemblesTests
Error:  testCrashBrokerWithoutCursorLedgerLeak(org.apache.pulsar.broker.service.BrokerBkEnsemblesTests)  Time elapsed: 2.499 s  <<< FAILURE!
java.lang.AssertionError: 
expected [null] but found [108,121,1641378459685,1641378459830,2,0,0,0,245,0,108
]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotSame(Assert.java:1033)
	at org.testng.Assert.assertNull(Assert.java:965)
	at org.testng.Assert.assertNull(Assert.java:953)
	at org.apache.pulsar.broker.service.BrokerBkEnsemblesTests.testCrashBrokerWithoutCursorLedgerLeak(BrokerBkEnsemblesTests.java:153)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by shankeerthan-kasilingam** [comment]
I would like to work on this issue.

**Comment by shibd** [comment]
+1 https://github.com/apache/pulsar/runs/4737886315?check_suite_focus=true

**Comment by lhotari** [comment]
recent failure: https://github.com/apache/pulsar/runs/4863165088?check_suite_focus=true#step:9:1084

**Comment by shibd** [comment]
@lhotari I reproduced it, I push a PR, PTAL.

**Comment by lhotari** [comment]
Fixed by #14519
